### DISCUSSION TITLE
Search in translated values

### DISCUSSION THREAD

**Discussion by peteror** [saleor/saleor#9284] [Q&A] [answered]
I have noticed that product search only works on the original product name and description fields, but not on the translated values. (I'm using the products query with the search filter) 
I tried to search the documentation, but could not find anything. Is there a way to extend search to the translated fields as well?

**Comment by ghost** [comment]
hi @karolkielecki, do we have support for this now ?

**Comment by peelar** [comment, marked as answer]
Hello @sameer-setia @peteror 👋 

I recommend inspecting our [Search App](https://github.com/saleor/apps/tree/main/apps/search) in the [saleor/apps](https://github.com/saleor/apps) repository. We don't index translations in our current Algolia integration, but overall [it should be possible](https://www.algolia.com/doc/guides/managing-results/optimize-search-results/handling-natural-languages-nlp/how-to/multilingual-search/#one-index-for-all-records).

Feel free to fork the app, submit [a feature request](https://github.com/saleor/apps/discussions), or a PR 😉.

**Comment by ghost** [comment]
hello @peelar 
when will this be included in saleor-core backend as in website it is mentioned that algolia search will be coming soon
![Screenshot from 2023-02-13 15-02-17](https://user-images.githubusercontent.com/83389170/218422619-a3d22697-2b63-4a50-96eb-26c16d1c7e10.png)
and is [Search App](https://github.com/saleor/apps/tree/main/apps/search) stable, can we use it in production environment?

**Reply by peelar** [reply]
@sameer-setia 

Algolia integration is an app, it's not going to be added to the core Saleor API.

All the apps from the [saleor/apps](https://github.com/saleor/apps) repository are currently available in the feature preview. This is a stage where we focus on collecting feedback and fundamentally aligning apps with the needs of our users. That also means the apps can still change pretty drastically. You can always [fork](https://github.com/saleor/apps/blob/main/docs/forking.md) apps and maintain their alternative version. 

Since you are especially interested in the Algolia integration, we would love if you shared your use case and told us what the Search App has to do to make your life easier. If you are interested, [Apps GitHub Discussions](https://github.com/saleor/apps/discussions) is the place to do it.
